operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -37,12 +37,14 @@ WDT_ROOT="/u01/wdt/weblogic-deploy"
 WDT_OUTPUT="/tmp/wdt_output.log"
 WDT_BINDIR="${WDT_ROOT}/bin"
 WDT_FILTER_JSON="/weblogic-operator/scripts/model_filters.json"
-WDT_CREATE_FILTER="/weblogic-operator/scripts/wdt_create_filter.py"
+WDT_CREATE_FILTER="/weblogic-operator/scripts/model_wdt_create_filter.py"
```

Comment:
Can we use "-" instead of "_" in the model script names? That seems to be the most common convention in operator file names that are using a separator char.  (BTW It's nice that you've made sure that all of the model related scripts now start with 'model'.)

Reply:
done